tests/installation/module_registration/add_ha_regcode.pm

```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: After register Application module
```

Comment:
I don't think this is related with Application module, it is just for HA that needs additional code.